probe/endpoint/reporter.go

```diff
@@ -71,34 +70,34 @@ func (r *Reporter) Report() (report.Report, error) {
 
 func (r *Reporter) addConnection(rpt *report.Report, c *procspy.Connection) {
 	var (
-		scopedLocal  = report.MakeAddressNodeID(r.hostID, c.LocalAddress.String())
-		scopedRemote = report.MakeAddressNodeID(r.hostID, c.RemoteAddress.String())
-		key          = report.MakeAdjacencyID(scopedLocal)
-		edgeKey      = report.MakeEdgeID(scopedLocal, scopedRemote)
+		localAddressNodeID  = report.MakeAddressNodeID(r.hostID, c.LocalAddress.String())
+		remoteAddressNodeID = report.MakeAddressNodeID(r.hostID, c.RemoteAddress.String())
+		adjacencyID         = report.MakeAdjacencyID(localAddressNodeID)
+		edgeID              = report.MakeEdgeID(localAddressNodeID, remoteAddressNodeID)
 	)
 
-	rpt.Address.Adjacency[key] = rpt.Address.Adjacency[key].Add(scopedRemote)
+	rpt.Address.Adjacency[adjacencyID] = rpt.Address.Adjacency[adjacencyID].Add(remoteAddressNodeID)
 
-	if _, ok := rpt.Address.NodeMetadatas[scopedLocal]; !ok {
-		rpt.Address.NodeMetadatas[scopedLocal] = report.NewNodeMetadata(map[string]string{
-			docker.Name: r.hostName,
-			docker.Addr: c.LocalAddress.String(),
```

Comment:
It actually doesn't make sense to define a constant for the "addr" literal, because we've been imprecise in how we [ab]use it throughout the code. We use it as a key to read, without knowing which component actually used it to write.

```fish
ugh ~/src/github.com/weaveworks/scope (consistent-variable-naming) ack '"addr"'
experimental/example/searchapp/app.go
22:             addr = flag.String("addr", ":8080", "HTTP listen address")

experimental/example/shout/shout.go
22:             addr = flag.String("addr", ":8090", "HTTP listen address")

probe/docker/tagger.go
13:     Addr        = "addr"

probe/endpoint/reporter.go
84:                     "addr": c.LocalAddress.String(),
103:                            "addr": c.LocalAddress.String(),

render/detailed_node.go
128:    local := fmt.Sprintf("%s:%s", nmd.Metadata["addr"], nmd.Metadata["port"])
153:    if val, ok := nmd.Metadata["addr"]; ok {

render/mapping.go
52:             id      = MakeEndpointID(report.ExtractHostID(m), m.Metadata["addr"], m.Metadata["port"])
53:             major   = fmt.Sprintf("%s:%s", m.Metadata["addr"], m.Metadata["port"])
112:            id    = MakeAddressID(report.ExtractHostID(m), m.Metadata["addr"])
113:            major = m.Metadata["addr"]

test/report_fixture.go
89:                                     "addr":            ClientIP,
95:                                     "addr":            ClientIP,
101:                                    "addr":            ServerIP,
213:                                    "addr":            ClientIP,
217:                                    "addr":            ServerIP,
```